docs/source/getting_started/dataset_creation/dataset_creation_local_upload.rst

```diff
@@ -0,0 +1,37 @@
+.. _dataset-creation-local-upload:
+
+Create dataset with local upload
```

Comment:
what's local upload? I would call it differently, e.g. simply "Create a dataset" or "Create a dataset from a local dataset"